os/pm/pm_suspendcount.c

```diff
@@ -77,25 +77,23 @@
  * Name: pm_suspendcount
  *
  * Description:
- *   This function is called to get current stay count.
+ *   This function is called to get current suspend count of domain.
  *
  * Input Parameters:
  *   domain_id - The domain ID of the PM activity
  *
  * Returned Value:
- *   Current pm stay count
- *
- * Assumptions:
- *   This function may be called from an interrupt handler.
+ *   non-negative interger: the suspend count of domain
+ *   ERROR: for invalid domain_id
  *
  ****************************************************************************/
 
-uint16_t pm_suspendcount(int domain_id)
+int pm_suspendcount(int domain_id)
```

Comment:
I think, Return type should be same uint16_t

Reply:
We need `int` type to return `ERROR`